playbooks/georchestra.yml

```diff
@@ -242,6 +242,9 @@
       port: 8780
       gitrepo: https://github.com/mviewer/mviewerstudio
       gitversion: master
+    gateway:
+      enabled: true
```

Comment:
unsure about it .. i'd leave that at `false` for now ? or revert the logic in the nginx template fragment to default to s-p unless disabled ?

until there's more testing.. who am i kidding, who uses this playbook besides us ? :)